### PR TITLE
Remove typed parameters from method signatures

### PR DESCRIPTION
**Pull request by ghost**
**TRAC-797**: https://jira.lib.utk.edu/browse/TRAC-797

# What does this Pull Request do?

Remove typed parameters from method signatures

# What's new?
A few words have been removed from the files:
includes/IslandoraFedoraObjectLabelCache.php
includes/IslandoraSimpleDSVCollectionPopulator.php
includes/SimpleDSVCollection.php

# How should this be tested?
replace the clone from github for the islandora scholar in the scripts/modules.sh file in TRACE.
Change the code from:
> git clone https://github.com/utkdigitalinitiatives/islandora_ingest_collections.git
to
> git clone https://github.com/robert-patrick-waltz/islandora_ingest_collections.git

also, need to checkout the correct branch, add this into the modules file after the  line:
> cd "$DRUPAL_HOME"/sites/all/modules/islandora_ingest_collections || exit
add

```
git fetch
git checkout TRAC-797

```
perform a vagrant destroy  and vagrant up. 
After the VM is provisioned completely, you should be able to see the page:

> http://localhost:8000/browse

with all nested collections.

In other words, the change is tested by a custom script, and if the script does not create those collection entries, then the change broke the code.

# Additional Notes:
Where is that confounded bridge?

# Interested parties
@CanOfBees @pc37utn

### CODE REVIEW

**Comment by pc37utn** [comment]
I did a vagrant rebuild and the collections show up, going to merge this
